docs/admin/dns.md

```diff
@@ -9,10 +9,13 @@ assignees:
 ## Introduction
 
 As of Kubernetes 1.3, DNS is a built-in service launched automatically using the addon manager [cluster add-on](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/README.md).
-A DNS Pod and Service will be scheduled on the cluster, and the kubelets will be
-configured to tell individual containers to use the DNS Service's IP to resolve DNS names.
 
-Every Service defined in the cluster (including the DNS server itself) will be
+A scheduled DNS Pod and Service on the cluster, and the kubelets configured to
```

Comment:
Looks like this is an incomplete sentence. How about:

"Kubernetes DNS schedules a DNS Pod and Service on th cluster, and configures the kubelets to tell individual containers to use the DNS Service's IP to resolve DNS names."